packages/twenty-front/codegen.cjs

```diff
@@ -1,4 +1,3 @@
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
I didn't comment on that because I didn't see the concern of having that in a codegen